python/src/DiscreteMarkovChain_doc.i.in

```diff
@@ -87,6 +87,17 @@ matrix : :class:`~openturns.SquareMatrix`
 %feature("docstring") OT::DiscreteMarkovChain::setOrigin
 "Accessor to the origin.
 
+Returns
+-------
+distribution : :class:`~openturns.UserDefined`
+    The stationary probability distribution of the Markov chain, i.e.
```

Comment:
I think you switched the descriptions of `setOrigin` and of `computeStationaryDistribution`.

Reply:
Right. Thanks!